sale_report_delivered/reports/sale_report.py

```diff
@@ -165,7 +165,7 @@ def _from(self):
 
     def _where(self):
         return """
-            WHERE sm.state = 'done' OR sm.state IS NULL
+            WHERE sm.to_refund = false AND (sm.state = 'done' OR sm.state IS NULL)
```

Comment:
For shorter form, but shouldn't be the other way (to include those to refund)?

```suggestion
            WHERE NOT sm.to_refund AND (sm.state = 'done' OR sm.state IS NULL)
```